shared/api/certificate.go

```diff
@@ -48,9 +48,9 @@ type CertificatesPost struct {
 	// API extension: certificate_self_renewal
 	Certificate string `json:"certificate" yaml:"certificate"`
 
-	// Server trust password (used to add an untrusted client)
+	// Trust token (used to add an untrusted client)
 	// Example: blah
-	Password string `json:"password" yaml:"password"`
+	TrustToken string `json:"trust_token" yaml:"trust_token"`
```

Comment:
What is this needed for? We can use tokens already today right?

Reply:
It is required if LXD's addresses in the token cannot be reached from the outside due to NAT, see the info box here: https://documentation.ubuntu.com/lxd/en/latest/authentication/#adding-client-certificates-using-tokens.

Reply:
Ah OK - we need an API extension for that then. Did Incus add one via a separate commit?